versioned_docs/version-7.x/development/05-tips.md

```
@@ -0,0 +1,21 @@
+# Tips
+This section details some tips on common development tasks/setup.
+
+## API
+
+1\. Developers may switch easily between sqlite, postgresql, and mariadb environment variables by using the scripts in /receipt-wrangler/api/dev. Use
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)